### PR TITLE
docs: Add biweekly-report.md(0731-0813)

### PR DESCRIPTION
**Pull request by Qizeng-api7**
Add biweekly-report.md(0731-0813)

### CODE REVIEW

File: blog/en/blog/2023/08/15/weekly-report.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+- Apache APISIX
+- API Gateway
+- Weekly Report
+- Contributor
+description: The Apache APISIX Community Biweekly Report can help community members better grasp the progress of the Apache APISIX community so that everyone can participate in the Apache APISIX community.
+tags: [Community]
+image: https://static.apiseven.com/uploads/2023/08/15/cOgKwDgK_%E8%8B%B1%E6%96%87%E5%A4%B4%E5%9B%BE.png
+---
+
+> We have recently made fixes and improvements to certain features of Apache APISIX. These include the addition of the `chaitin-waf` plugin, support for configuring TLS handshake protocol features for specified SNI, YAML configuration file support for rendering configurations from environment variables, and ensuring that the `Limit-count` plugin returns the correct X-RateLimit-Reset when rejecting requests. For more details, please refer to the current bi-weekly report.
```
Comment (Yilialinn) [inline blog/en/blog/2023/08/15/weekly-report.md]:
```suggestion
> We have recently made fixes and improvements to certain features of Apache APISIX. These include supporting the `chaitin-waf` plugin, configuring TLS handshake protocol for specified SNI, configuring YAML file for rendering environment variables, and ensuring that the `limit-count` plugin returns the correct X-RateLimit-Reset when rejecting requests. For more details, please refer to the biweekly report.
```

Comment (yzeng25) [inline blog/en/blog/2023/08/15/weekly-report.md]:
```suggestion
> We have recently made fixes and improvements to certain features of Apache APISIX. These include the addition of the `chaitin-waf` plugin, support for configuring TLS handshake protocol features for specified SNI, YAML configuration file support for rendering configurations from environment variables, and ensuring that the `limit-count` plugin returns the correct X-RateLimit-Reset when rejecting requests. For more details, please refer to the current bi-weekly report.
```

File: blog/en/blog/2023/08/15/weekly-report.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+### Apache APISIX
+
+- [Add `chaitin-waf` plugin](https://github.com/apache/apisix/pull/9838) (Contributor: [lingsamuel](https://github.com/lingsamuel))
+
+- [Add support for configuring TLS handshake protocol for specified SNI](https://github.com/apache/apisix/pull/9903) (Contributor: [AlinsRan](https://github.com/AlinsRan))
+
+- [YAML configuration file supports rendering configurations from environment variables](https://github.com/apache/apisix/pull/9855) (Contributor: [shreemaan-abhishek](https://github.com/shreemaan-abhishek))
+
+- [`Limit-count` plugin returns the correct X-RateLimit-Reset when rejecting requests](https://github.com/apache/apisix/pull/9978) (Contributor: [jiangfucheng](https://github.com/jiangfucheng))
```
Comment (yzeng25) [inline blog/en/blog/2023/08/15/weekly-report.md]:
```suggestion
- [`limit-count` plugin returns the correct X-RateLimit-Reset when rejecting requests](https://github.com/apache/apisix/pull/9978) (Contributor: [jiangfucheng](https://github.com/jiangfucheng))
```

File: blog/zh/blog/2023/08/02/15/weekly-report.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+- Apache APISIX
+- API 网关
+- 社区周报
+- 贡献者
+description: “独行者速，众行者远”。Apache APISIX 社区周报希望可以帮助社区小伙伴们更好地掌握 Apache APISIX 社区的进展，方便大家参与到 Apache APISIX 社区中来。
+tags: [Community]
+image: https://static.apiseven.com/uploads/2023/08/15/onL0VVoC_%E4%B8%AD%E6%96%87%E5%A4%B4%E5%9B%BE.png
+---
+
+> 我们近期对 Apache APISIX 的部分功能进行了修复或改善，包括增加 `chaitin-waf` 插件、增加为指定 SNI 配置 TLS 握手协议特性支持、YAML 配置文件支持从环境变量渲染配置、`Limit-count` 插件在拒绝请求时返回正确的 X-RateLimit-Reset，详情请阅读本期双周报。
```
Comment (yzeng25) [inline blog/zh/blog/2023/08/02/15/weekly-report.md]:
```suggestion
> 我们近期对 Apache APISIX 的部分功能进行了修复或改善，包括增加 `chaitin-waf` 插件、增加为指定 SNI 配置 TLS 握手协议特性支持、YAML 配置文件支持从环境变量渲染配置、`limit-count` 插件在拒绝请求时返回正确的 X-RateLimit-Reset，详情请阅读本期双周报。
```

File: blog/zh/blog/2023/08/02/15/weekly-report.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+我们近期对部分功能进行了修复或改善，更新内容汇总如下：
+
+1. 增加 `chaitin-waf` 插件
+
+2. 增加为指定 SNI 配置 TLS 握手协议特性支持
+
+3. YAML 配置文件支持从环境变量渲染配置
+
+4. `Limit-count` 插件在拒绝请求时返回正确的 X-RateLimit-Reset
```
Comment (yzeng25) [inline blog/zh/blog/2023/08/02/15/weekly-report.md]:
```suggestion
4. `limit-count` 插件在拒绝请求时返回正确的 X-RateLimit-Reset
```

File: blog/zh/blog/2023/08/02/15/weekly-report.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+### Apache APISIX
+
+- [增加 `chaitin-waf` 插件](https://github.com/apache/apisix/pull/9838)（贡献者：[lingsamuel](https://github.com/lingsamuel)）
+
+- [增加为指定 SNI 配置 TLS 握手协议特性支持](https://github.com/apache/apisix/pull/9903)（贡献者：[AlinsRan](https://github.com/AlinsRan)）
+
+- [YAML 配置文件支持从环境变量渲染配置](https://github.com/apache/apisix/pull/9855)（贡献者：[shreemaan-abhishek](https://github.com/shreemaan-abhishek)）
+
+- [`Limit-count` 插件在拒绝请求时返回正确的 X-RateLimit-Reset](https://github.com/apache/apisix/pull/9978)（贡献者：[jiangfucheng](https://github.com/jiangfucheng)）
```
Comment (yzeng25) [inline blog/zh/blog/2023/08/02/15/weekly-report.md]:
```suggestion
- [`limit-count` 插件在拒绝请求时返回正确的 X-RateLimit-Reset](https://github.com/apache/apisix/pull/9978)（贡献者：[jiangfucheng](https://github.com/jiangfucheng)）
```

File: blog/zh/blog/2023/08/02/15/weekly-report.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+  我们很高兴地宣布 Apache APISIX 3.4.1 版本已经发布，其中包含了针对 JWT 的安全补丁。
+
+- [APISIX 的出海征程势如破竹：APISIX 马来西亚 Meetup](https://apisix.apache.org/zh/blog/2023/07/12/2023-apisix-meetup-malaysia/)
+
+    近日，Apache APISIX 发起的技术盛宴，在马来西亚首都吉隆坡隆重开幕！活动由 API7（支流科技）与 N2N（N2N Connect Berhad，联圣）及 AMD（Advanced Micro Devices, Inc.,）联合举办，汇集了众多 API 网关技术领域的专家和爱好者。
+
+- [Release Apache APISIX 3.4.0](https://apisix.apache.org/zh/blog/2023/06/30/release-apache-apisix-3.4.0/)
+  
+    此版本提供了一个新的插件 loki-logger，可将日志转发到 Grafana Loki，并允许在路由级别上建立 mTLS 连接。此外，此版本还还引入了许多新的特性，旨在优化对 APISIX 的使用体验。
```
Comment (yzeng25) [inline blog/zh/blog/2023/08/02/15/weekly-report.md]:
```suggestion
    此版本提供了一个新的插件 `loki-logger`，可将日志转发到 Grafana Loki，并允许在路由级别上建立 mTLS 连接。此外，此版本还还引入了许多新的特性，旨在优化对 APISIX 的使用体验。
```

File: blog/en/blog/2023/08/15/weekly-report.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+
+Apache APISIX grew up as a community from the first day it was open-sourced, and quickly became the most active open-source API gateway project in the world. These achievements are inseparable from the joint efforts of community partners.
+
+From 7.31 to 8.13, 20 contributors submitted 37 commits for Apache APISIX. Thank you for your contributions to Apache APISIX.
+
+We have recently fixed and improved some features, and the summary of the updates is as follows:
+
+1. Support `chaitin-waf` plugin
+
+2. Support host level dynamic setting of tls protocol version
```
Comment (Yilialinn) [inline blog/en/blog/2023/08/15/weekly-report.md]:
```suggestion
2. Support host-level dynamic setting of TLS protocol version
```

File: blog/en/blog/2023/08/15/weekly-report.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+
+3. YAML configuration file supports rendering configurations from environment variables
+
+4. X-RateLimit-Reset shouldn't be set to 0 after request be rejected
```
Comment (Yilialinn) [inline blog/en/blog/2023/08/15/weekly-report.md]: Revise accordingly to the first paragraph

File: blog/en/blog/2023/08/15/weekly-report.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+  
+  In this article, we will look at examples of how we can use the rate limiting plugins in APISIX. You can find the complete configuration files and instructions to deploy for this article in this repository.
+
+- [Release Apache APISIX 3.4.1](https://apisix.apache.org/blog/2023/07/21/release-apache-apisix-3.4.1/)
+
+  We are pleased to present Apache APISIX 3.4.1 with a security patch for JWT.
+
+- [Accelerating API Gateway Excellence: Apache APISIX Community Meetup in Malaysia](https://apisix.apache.org/blog/2023/07/11/2023-apisix-meetup-malaysia/)
+  
+  Kuala Lumpur, the capital of Malaysia, witnessed the success of the 2023 APISIX open-source Community Meetup on July 4th.
```
Comment (Yilialinn) [inline blog/en/blog/2023/08/15/weekly-report.md]: Change this one to https://apisix.apache.org/blog/2023/07/20/data-mask-plugin/

File: blog/en/blog/2023/08/15/weekly-report.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+
+## Recent Blog Recommendations
+
+- [Rate Limit Your APIs With Apache APISIX](https://apisix.apache.org/blog/2023/08/14/rate-limit/)
+  
+  In this article, we will look at examples of how we can use the rate limiting plugins in APISIX. You can find the complete configuration files and instructions to deploy for this article in this repository.
+
+- [Release Apache APISIX 3.4.1](https://apisix.apache.org/blog/2023/07/21/release-apache-apisix-3.4.1/)
+
+  We are pleased to present Apache APISIX 3.4.1 with a security patch for JWT.
```
Comment (Yilialinn) [inline blog/en/blog/2023/08/15/weekly-report.md]: Change this one to https://apisix.apache.org/blog/2023/07/23/release-apache-apisix-3.2.2/

File: blog/zh/blog/2023/08/02/15/weekly-report.md (hunk 3 above)
Comment (Yilialinn) [inline blog/zh/blog/2023/08/02/15/weekly-report.md]: Don't need to uppercase "l" in "limit-count". Please check the whole passage.

File: blog/zh/blog/2023/08/02/15/weekly-report.md (hunk 6 above)
Comment (Yilialinn) [inline blog/zh/blog/2023/08/02/15/weekly-report.md]: Change these to
1. 3.2.2
2. 3.4.1
3. meetup

File: blog/zh/blog/2023/08/02/15/weekly-report.md (hunk 5 above)
Comment (SylviaBABY) [inline blog/zh/blog/2023/08/02/15/weekly-report.md]:
```suggestion
- [`Limit-count` 插件在拒绝请求时返回正确的 X-RateLimit-Reset 数据](https://github.com/apache/apisix/pull/9978)（贡献者：[jiangfucheng](https://github.com/jiangfucheng)）
```

File: blog/zh/blog/2023/08/02/15/weekly-report.md [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+
+从 2023.7.31 - 2023.8.13，有 20 位开发者为 Apache APISIX 提交了 37 个 commit。感谢这些小伙伴为 Apache APISIX 添砖加瓦，是你们的无私付出，让 Apache APISIX 项目变得更好！
+
+我们近期对部分功能进行了修复或改善，更新内容汇总如下：
+
+1. 增加 `chaitin-waf` 插件
+
+2. 增加为指定 SNI 配置 TLS 握手协议特性支持
+
+3. YAML 配置文件支持从环境变量渲染配置
```
Comment (SylviaBABY) [inline blog/zh/blog/2023/08/02/15/weekly-report.md]:
```suggestion
3. apisix.yaml 配置文件支持从环境变量渲染配置
```

File: blog/zh/blog/2023/08/02/15/weekly-report.md [hunk 12]
```diff
@@ -0,0 +1,67 @@
...
+
+## 近期亮点功能
+
+### Apache APISIX
+
+- [增加 `chaitin-waf` 插件](https://github.com/apache/apisix/pull/9838)（贡献者：[lingsamuel](https://github.com/lingsamuel)）
+
+- [增加为指定 SNI 配置 TLS 握手协议特性支持](https://github.com/apache/apisix/pull/9903)（贡献者：[AlinsRan](https://github.com/AlinsRan)）
+
+- [YAML 配置文件支持从环境变量渲染配置](https://github.com/apache/apisix/pull/9855)（贡献者：[shreemaan-abhishek](https://github.com/shreemaan-abhishek)）
```
Comment (SylviaBABY) [inline blog/zh/blog/2023/08/02/15/weekly-report.md]: ditto

File: blog/en/blog/2023/08/15/weekly-report.md [hunk 13]
```diff
@@ -0,0 +1,67 @@
...
+
+From 7.31 to 8.13, 20 contributors submitted 37 commits for Apache APISIX. Thank you for your contributions to Apache APISIX.
+
+We have recently fixed and improved some features, and the summary of the updates is as follows:
+
+1. Support `chaitin-waf` plugin
+
+2. Support host level dynamic setting of tls protocol version
+
+3. YAML configuration file supports rendering configurations from environment variables
```
Comment (SylviaBABY) [inline blog/en/blog/2023/08/15/weekly-report.md]:
```suggestion
3. apisix.yaml configuration supports rendering configurations from environment variables
```

File: blog/en/blog/2023/08/15/weekly-report.md [hunk 14]
```diff
@@ -0,0 +1,67 @@
...
+
+## Highlights of Recent Features
+
+### Apache APISIX
+
+- [Add `chaitin-waf` plugin](https://github.com/apache/apisix/pull/9838) (Contributor: [lingsamuel](https://github.com/lingsamuel))
+
+- [Add support for configuring TLS handshake protocol for specified SNI](https://github.com/apache/apisix/pull/9903) (Contributor: [AlinsRan](https://github.com/AlinsRan))
+
+- [YAML configuration file supports rendering configurations from environment variables](https://github.com/apache/apisix/pull/9855) (Contributor: [shreemaan-abhishek](https://github.com/shreemaan-abhishek))
```
Comment (SylviaBABY) [inline blog/en/blog/2023/08/15/weekly-report.md]: ditto

**Review by yzeng25 — DISMISSED** [review]

**Review by Yilialinn — APPROVED** [review]

**Review by guoqqqi — DISMISSED** [review]

**Comment by guoqqqi** [comment]
image
The file directory looks a bit wrong

**Comment by yzeng25** [comment]
> image The file directory looks a bit wrong

@Qizeng-api7 the folder name should be `zh/blog/2023/08/15`.

**Review by SylviaBABY — APPROVED** [review]

**Review by guoqqqi — APPROVED** [review]

**Pull request merged**